frmChanThreadWatch.cs

```diff
@@ -419,8 +420,12 @@ public partial class frmChanThreadWatch : Form {
                             });
                         }
                         else {
-                            Process.Start(dir);
-                        }
+							if(RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) {
```

Comment:
Just noticed that the rest of the file has spaces - let me know and I'll change the formatting on my end

Reply:
Yes, you can change this as well as add a space after the `if` keyword for consistency.